spring-boot/config-server/src/main/java/eu/coatrack/admin/server/service/filter/CheckIfUrlAndCredentialsFitWithEachOtherFilter.java

```diff
@@ -42,24 +42,19 @@
     private final String CREDENTIAL_URI = "/credentials";
 
     @Override
-    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
-        if (request instanceof HttpServletRequest
+    protected void doFilterInternal(HttpServletRequest servletRequest, HttpServletResponse servletResponse, FilterChain filterChain) throws ServletException, IOException {
+        if (!servletResponse.isCommitted()
                 // added the following check to avoid "IllegalStateException: resource already commited"
-                && !response.isCommitted()) {
-            /**
-             * Cast request/response
-             */
-            HttpServletRequest httpRequest = (HttpServletRequest) request;
-            HttpServletResponse httpResponse = (HttpServletResponse) response;
+                 ) {
```

Comment:
Bad formatting. This should be right behind the if-condition.

Reply:
Now I noticed that this is not related to this PR. Maybe create a follow up issue to correct the formating?

Reply:
This is such a small thing. An own issue would over the top. I think you should just fix it here.

Reply:
This is solved